### DISCUSSION TITLE
Getting involved -- express your interest

### DISCUSSION THREAD

**Discussion by woodseowl** [CornellCustomDev/CD-LaravelStarterKit#3] [Polls]
See where you think you'd fall on this scale. It's not a commitment, so just go with whatever seems like the best fit.

And if you'd like to be involved, maybe share that in a comment?

**Comment by woodseowl** [comment]
I'm planning to get the ball rolling on this, hoping to get at least one other dev and a designer who can work on it strategically with me to clarify requirements and plan out the features.

**Comment by sg667** [comment]
So I responded with the conversations role (mostly because my experience with Laravel is relatively minor).  I would love to be involved with helping to implement accessible components though, be it through consultation or tweaking an existing foundation.
